aspnetcore/toc.md

```diff
@@ -196,7 +196,7 @@
 #### [Publish to a Docker Image](https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker/)
 ## [Deployment](xref:tutorials/publish-to-azure-webapp-using-vs)
 ### [Publish to Azure with Visual Studio](xref:tutorials/publish-to-azure-webapp-using-vs)
-### [Publish to Azure with CLI tools](xref:tutorials/publish-to-azure-webapp-using-cli)
+### [Publish to Azure with CLI tools](tutorials/publish-to-azure-webapp-using-cli.md)
```

Comment:
Can you use xref:
They are more robust than relative paths. You can copy/paste them.